Mathlib/Analysis/Calculus/ContDiff/Basic.lean

```diff
@@ -274,6 +274,13 @@ theorem ContinuousLinearEquiv.iteratedFDerivWithin_comp_left (g : F ≃L[𝕜] G
       ContinuousLinearMap.compContinuousMultilinearMap_coe, EmbeddingLike.apply_eq_iff_eq]
     rw [iteratedFDerivWithin_succ_apply_left]
 
+/-- Iterated derivatives commute with left composition by continuous linear equivalences- -/
+theorem ContinuousLinearEquiv.iteratedFDeriv_comp_left {f : E → F} {x : E} (g : F ≃L[𝕜] G) {i : ℕ} :
+    iteratedFDeriv 𝕜 i (g ∘ f) x =
+      (g : F →L[𝕜] G).compContinuousMultilinearMap (iteratedFDeriv 𝕜 i f x) := by
```

Comment:
Does writing `g.toContinuousLinearMap` work?

Reply:
Yes, that works. Replaced `(g : F →L[𝕜] G)` by `g.toContinuousLinearMap` as suggested.